### DISCUSSION TITLE
Incorporating QUDT into my app; adding new units

### DISCUSSION THREAD

**Discussion by chipmasters** [qudt/qudt-public-repo#558] [Q&A] [answered]
I am interested in using the QUDT units dataset in the HHEAR Data Center harmonized data repository. (c.f. https://hheardatacenter.mssm.edu/). However, there are several units that we need to use that are not currently in the QUDT Units of Measure for All Units Release 2.1.19 dataset. My question is what is the process for adding (or requesting to have added) new units to the dataset? And, as a follow-up, do you support, or define any best practices, for users to keep local copies of prospective units definitions, following the QUDT schema, so that we can use them prior to their release?

Finally, I imagine we have some needs for units that may not be considered appropriate (or maybe so?) to include in the general QUDT dataset. Do you have any past experience with other users simply using the QUDT schema model to maintain their own datasets of units that you don't include in the official releases? (For example, we have units like "cigarettes per day" and "endotoxin unit per cubic meter")

UPDATE: I read the documentation and found the page on units vocabulary submission, so I think it answers most, if not all, of my questions. So I think I am good.

**Comment by jhodgesatmb** [comment, marked as answer]
We are committed to supporting the units (etc.) covered by standards such
as ISO 80000, but there are specialized domain units that need to be
covered that either haven't made it into standards yet or may never. As a
result they need to be curated by subject matter experts such as you. For
this we have a submission guideline:

On the qudt.org landing page there is a block of links on the right, and
one of them points to the unit submission guidelines. I have attached a pic
showing the path.


On Tue, Sep 6, 2022 at 11:09 AM chipmasters ***@***.***>
wrote:

> I am interested in using the QUDT units dataset in the HHEAR Data Center
> harmonized data repository. (c.f. https://hheardatacenter.mssm.edu/).
> However, there are several units that we need to use that are not currently
> in the QUDT Units of Measure for All Units Release 2.1.19 dataset. My
> question is what is the process for adding (or requesting to have added)
> new units to the dataset? And, as a follow-up, do you support, or define
> any best practices, for users to keep local copies of prospective units
> definitions, following the QUDT schema, so that we can use them prior to
> their release?
>
> Finally, I imagine we have some needs for units that may not be considered
> appropriate (or maybe so?) to include in the general QUDT dataset. Do you
> have any past experience with other users simply using the QUDT schema
> model to maintain their own datasets of units that you don't include in the
> official releases? (For example, we have units like "cigarettes per day"
> and "endotoxin unit per cubic meter")
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/qudt/qudt-public-repo/discussions/558>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AATQRWOQMXOSW43UFGHMXT3V46CETANCNFSM6AAAAAAQGCCEOY>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Jack

**Comment by chipmasters** [comment]
Thanks Jack. I also found the submission guidelines page after I posted the
original discussion. Appreciate you following up. I’ll work on creating a
dataset file with the new units I need, which are mainly related to the
biomedical and lab science domain, and submit a pull request. If I have any
questions about specific units, I’ll post to this thread.

Best,
Chip
